custom/src/main/java/com/splunk/opentelemetry/logging/SplunkSlf4jLoggingCustomizer.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.opentelemetry.logging;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.logging.simple.Slf4jSimpleLoggingCustomizer;
+import io.opentelemetry.javaagent.tooling.LoggingCustomizer;
+
+@AutoService(LoggingCustomizer.class)
+public class SplunkSlf4jLoggingCustomizer implements LoggingCustomizer {
+  private final Slf4jSimpleLoggingCustomizer delegate = new Slf4jSimpleLoggingCustomizer();
+
+  @Override
+  public String name() {
+    return delegate.name();
```

Comment:
There are now 2 `LoggingCustomizer`s with the same name. Is it guaranteed that ours will always be the one that runs? Should we perhaps exclude the original one from the SPI file?

Reply:
Currently all our services are listed before upstream services in the service files.
Excluding it seems like fine idea, but actually doing it will probably require an unhealthy amount of gradle magic. I think we would need build a transformer as described in https://imperceptiblethoughts.com/shadow/configuration/merging/#merging-service-descriptor-files WDYT is there a simpler way to do this? Should I attempt the transformer approach?